lang/c++/include/avro/Node.hh

```diff
@@ -25,6 +25,7 @@
 #include <cassert>
 #include <memory>
 #include <utility>
+#include <vector>
```

Comment:
Is this include used ?

Reply:
Yes, in line 63: `const std::vector<std::string> &aliases() const;`
Some compiler does not fails because definition is declare elsewhere. But not on my case.

Reply:
Got it! Thanks!
Please tale a look at https://github.com/apache/avro/pull/3053#pullrequestreview-2242266809